addons/sys_zeus/XEH_postInit.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+
+private _category = format ["ACRE2 %1", localize "str_a3_cfghints_curator_curator_displayname"];
+
+[_category, "ZeusTalkFromCamera",  [(LLSTRING(SpeakFromCamera)), (LLSTRING(SpeakFromCamera_description))],
+    { call FUNC(handleZeusSpeakPress) },
+    { call FUNC(handleZeusSpeakPressUp) },
+[41, [false, false, false]]] call CBA_fnc_addKeybind; //Default bound to `
```

Comment:
Can probably come up with a better default. Zeus doesn't use as many keys after-all.

Reply:
My group uses this almost every single OP now. In my opinion, it's a pretty good key.

Reply:
On my keyboard that's AltGr+7, rather hard to access.